opentreemap/treemap/js/src/lib/addResourceMode.js

```diff
@@ -120,11 +120,15 @@ function init(options) {
         U.$find('select', $form).on('change', enableFinalStep);
     }
```

Comment:
Not sure if this is significant, but **Add a Bioswale** causes lines 118 and 120 to generate warnings in `utility.js` that their respective selectors are not found.